AutonomousDemo/VehicleMotion.cs

```diff
@@ -10,24 +10,34 @@ public class VehicleMotion
     {
```

Comment:
Okay, I think I'm confused. Is this class supposed to represent a specific _chunk_ of movement of a vehicle over time, or is it supposed to represent the vehicle movement over time more generally?